diff -up vboxgtk-0.5.0/data/vboxgtk.desktop.in.orig vboxgtk-0.5.0/data/vboxgtk.desktop.in
--- vboxgtk-0.5.0/data/vboxgtk.desktop.in.orig	2009-08-09 18:19:46.370336267 +0100
+++ vboxgtk-0.5.0/data/vboxgtk.desktop.in	2009-08-09 20:59:52.904335950 +0100
@@ -1,10 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
+Version=1.0
 Name=VBoxGtk
 Comment=VirtualBox GTK+ interface
 Exec=vboxgtk
 Terminal=false
 StartupNotify=true
-Icon=vboxgtk
+Icon=/usr/share/icons/hicolor/scalable/apps/vboxgtk.svg
 Type=Application
-Categories=GTKi;Office;
+Categories=Emulator;System;
+X-Desktop-File-Install-Version=0.15
